Input/Input.js

```diff
@@ -269,7 +269,7 @@ const InputPopupBase = kind({
 	handlers: {
 		onShow: handle(
 			forward('onShow'),
-			(ev, {type}) => type === 'text' || type === 'password',
+			(ev, {type}) => type === 'text' || type === 'password' || type === 'url',
```

Comment:
Is this equivalent to below?

```suggestion
			(ev, {type}) => !type.includes('number'),
```

Reply:
Yes, and It looks simpler:) I fixed it as you suggested.